src/families/cosmos/Delegations/index.js

```diff
@@ -208,6 +220,59 @@ export default function Delegations({ account }: Props) {
                 },
               ]
             : []),
+          ...(undelegation
+            ? [
+                {
+                  label: t("cosmos.delegation.drawer.completionDate"),
+                  Component: () => (
+                    <LText numberOfLines={1} semiBold>
+                      <DateFromNow
+                        date={+new Date(undelegation.completionDate)}
+                      />
+                    </LText>
+                  ),
+                },
+              ]
+            : []),
+          ...(redelegation
+            ? [
+                {
+                  label: t("cosmos.delegation.drawer.redelegatedFrom"),
+                  Component: () => {
```

Comment:
please remove this pattern, creating a `() => {` component on the fly in a render() of a component is something to absolutely avoid.

from what i can tell your usecase is, you do not need a component at all, you need just to pass a React Element, not a React Component.

I would suggest to change

`Component: string | ComponentType<{}>,`


to 

`content: React$Node`

React$Node covers all you need: it can be a string, null, an element.

so basically on this side you would just do:

```js
content: <Touchable
                        onPress={() =>
                          onOpenExplorer(redelegation.validatorSrcAddress)
                        }
                        event="DelegationOpenExplorer"
                      >
                        <LText
                          numberOfLines={1}
                          semiBold
                          ellipsizeMode="middle"
                          style={[
                            drawerStyles.valueText,
                            drawerStyles.valueTextTouchable,
                          ]}
                        >
                          {redelegation.validatorSrcAddress}
                        </LText>
                      </Touchable>
```

and if you ever need a more complex component that you do not want to inline, you can make it simply a 


```js
<RedelegatedFrom redelegation={redelegation} />
```

but at least, the component refs don't change and the use of props makes React efficiently reupdating what's necessary.